scripts/qa_crowbarsetup.sh

```diff
@@ -4676,8 +4676,14 @@ function onadmin_prepare_cloudupgrade_admin_repos_6_to_7
     safely createrepo $tftpboot_repos_dir/PTF
 
     # change system repositories to SP2
```

Comment:
with the change below this comment becomes a bit misleading - maybe remove it as the code is rather clear - or have two copies within the 'if's